.pre-commit-config.yaml

```diff
@@ -1,11 +1,10 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
-- repo: https://github.com/dragomirecky/cmake-format-pre-commit-hook
-  rev: 'v0.6.0'
+- repo: https://github.com/cheshirekow/cmake-format-precommit
+  rev: v0.6.13
   hooks:
-    - id: cmake-format  # cmake formatter
-      files: (^|/)(CMakeLists.*|.*\.cmake|.*\.cmake.in)$
+    - id: cmake-format
```

Comment:
the files pattern, which has been here before, is no longer needed with the new cmake-format?

Reply:
No 👍